### DISCUSSION TITLE
Error reporting in production

### DISCUSSION THREAD

**Discussion by yonathan06** [tauri-apps/tauri#4935] [Q&A]
What would be the best way for error reporting on the Rust code?
When an exception is triggered in the browser, the app doesn't crash, so it is possible to send the error object and context to an error service.
But the main process running is Rust exists when the code panics, is there a way to send a report to a custom service when that happens? what would consider that best practice in that case?

**Comment by FabianLars** [comment]
I'm really not an expert on that topic, but i'd suggest checking out this cool project https://github.com/timfish/sentry-tauri

**Reply by yonathan06** [reply]
Thanks! I'm using Bugsnag, but I will definitely look at how it is implemented, I think it has the same principles
